paddle/fluid/framework/var_type_traits.h

```diff
@@ -140,6 +141,7 @@ using VarTypeRegistry = detail::VarTypeRegistryImpl<
     Tensor, LoDTensor, SelectedRows, std::vector<Scope *>, LoDRankTable,
     LoDTensorArray, platform::PlaceList, ReaderHolder, std::string, Scope *,
     std::map<size_t, Tensor>, operators::reader::LoDTensorBlockingQueueHolder,
+    std::shared_ptr<operators::reader::MultiDeviceLoDTensorBlockingQueueHolder>,
```

Comment:
how aboud using `MultiDeviceLoDTensorBlockingQueueHolder` instead of `LoDTensorBlockingQueueHolder `, we only need one blocking queue.